src/main/java/com/purbon/kafka/topology/roles/rbac/RBACBindingsBuilder.java

```diff
@@ -253,18 +253,20 @@ public String toString() {
 
     // Kafka Cluster scope
     List<String> topics =
-        Arrays.asList(
-            ksqlServer.commandTopic(),
-            ksqlServer.processingLogTopic(),
-            ksqlServer.consumerGroupPrefix());
+        Arrays.asList(ksqlServer.processingLogTopic(), ksqlServer.consumerGroupPrefix());
     for (String topic : topics) {
       TopologyAclBinding binding =
           apiClient.bind(ksqlServer.getPrincipal(), RESOURCE_OWNER, topic, LITERAL);
       bindings.add(binding);
     }
-    String resource = String.format("_confluent-ksql-%stransient", clusterId);
+
     TopologyAclBinding binding =
-        apiClient.bind(ksqlServer.getPrincipal(), RESOURCE_OWNER, resource, "Topic", PREFIX);
+        apiClient.bind(
+            ksqlServer.getPrincipal(), RESOURCE_OWNER, ksqlServer.internalTopics(), PREFIX);
```

Comment:
I am reviewed the [ksqldb-rbac-doc](https://docs.confluent.io/platform/current/security/rbac/ksql-rbac.html) trying to find a good explanation or reason for this permission, but I have not found.
Have I missed anything?  The previous link is the master role definition, so I take it always as a reference.

Reply:
Looks like RBAC ACLs are bit different and therefore we only need command topic acl. For that reason I reverted the changes I did and left commandTopic method to KsqlServerInstance.

Reply:
👍